### ISSUE TITLE
Warning: Creating default object from empty value

### ISSUE THREAD

**Issue by gilbitron** [Codeinwp/nivo-slider-lite#1]
``` php
Warning: Creating default object from empty value in .../wp-content/plugins/nivo-slider-lite/includes/core/includes/edd_sl_plugin_updater.php on line 86
```

**Comment by chrisdrake** [comment]
I have been experiencing this issue also. I notified the developers several weeks ago but still no fix has been incorporated.

**Comment by suture** [comment]
+1

**Comment by kathatp** [comment]
Same here

**Comment by gilbitron** [comment]
Fixed in v1.0.1
